sdk/src/Core/Amazon.Runtime/Credentials/Internal/_bcl+netstandard/SSOTokenManager.cs

```diff
@@ -484,7 +485,8 @@ public async Task<SsoToken> GetTokenAsync(SSOTokenManagerGetTokenOptions options
                     }
                     catch (Exception ex)
                     {
-                        _logger.Error(ex, $"Refreshing SSOToken for [{options.Session}] failed: {ex.Message}");
+                        // Exception message from SSOIDC client has text along with HTTP Body as JSON string.
+                        _logger.Error(ex, $"Refreshing SSOToken for [{options.Session}] failed: {ex.Message.Replace("{", "{{").Replace("}", "}}")}");
```

Comment:
There's also a `Refreshing SSOToken for [{options.Session}]` message in the `GetToken` (non-async) method: https://github.com/aws/aws-sdk-net/blob/9098bff8675d7a0be851e8d6056abb59351ce80b/sdk/src/Core/Amazon.Runtime/Credentials/Internal/_bcl%2Bnetstandard/SSOTokenManager.cs#L224

If the error message is the same you'll need the replace there too, right?

Reply:
Done